app/src/api/identity-verification.js

```diff
@@ -39,15 +34,15 @@ export const getVerificationStatus = async () => {
       });
 }
 
-export const initiateVerification = async () => {
+export const initiateVerification = async (http) => {
```

Comment:
I think it would make sense to expose `http` as a JS module as well so that we won't have to pass this as an arg from a react component.

I'll create an issue for this.

Reply:
Tracker: https://github.com/asgardeo/javascript/issues/128